pom.xml

```diff
@@ -20,7 +20,7 @@
 
     <groupId>com.ibm.websphere.azure</groupId>
     <artifactId>azure.websphere-traditional.singleserver</artifactId>
-    <version>1.0.38</version>
+    <version>1.0.39</version>
```

Comment:
Well, the offer in preview shows 1.0.40:

<img width="862" height="1792" alt="image" src="https://github.com/user-attachments/assets/aca0467b-dd27-4d1e-a9b6-41e34c91d9f2" />


So how can this be 1.0.39?